spec/core/terraform/config/base_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe Terraform::Config::Base do
+  let(:config) { described_class.new }
+
+  describe "#to_tf" do
+    subject(:to_tf) { config.to_tf }
```

Comment:
Why do we use `subject` instead of just doing:

```
expect { config.to_tf }.to raise_error(NotImplementedError)
```

Reply:
I prefer to use `subject`'s so that code won't repeat if other `it` blocks are added in the future